Utilizr/Util/DomainHelper.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Utilizr.Util
+{
+    public static class DomainHelper
+    {
+        public static List<string> GetSubDomain(string? fullurl)
+        {
+            List<string> subdomains = new List<string>();
+            string host = "";
+            string absolutepath = "";
+            GetDomain(fullurl, out host, out absolutepath);
+
+            subdomains.Add(host);
+            subdomains.Add($"*.{host}");
+
+            return subdomains;
+        }
+
+        public static bool GetDomain(string? url, out string host, out string absolutepath)

```

Comment:
I don't understand why we need the absolute path for a function trying to get the domain. It would be much simpler if the return type was just `string?` without the out parameters.